tests/README.md

```diff
@@ -0,0 +1,65 @@
+# Introduction
+
+The JSON-LD Test Suite is a set of tests that can
+be used to verify JSON-LD Processor conformance to the set of specifications
+that constitute JSON-LD. The goal of the suite is to provide an easy and
+comprehensive JSON-LD testing solution for developers creating JSON-LD Processors.
+
+More information and an RDFS definition of the test vocabulary can be found at [vocab](https://w3c.github.io/json-ld-api/tests/vocab).
+
+# Design
+
+Tests driven from a top-level [manifest](manifest.jsonld) and are defined for [stream-toRdf](stream-toRdf-manifest.jsonld):
+
+* [stream-toRdf](stream-toRdf-manifest.jsonld) tests have _input_ and _expected_ documents.
+  The _expected_ results can be compared using [RDF Dataset Isomorphism](https://www.w3.org/TR/rdf11-concepts/#dfn-dataset-isomorphism).
+
+  Stream-ToRdf tests may have a `expandContext` option, which is treated
+  as an IRI relative to the manifest.
+
+  For *NegativeEvaluationTests*, the result is a string associated with the expected error code.
+
+Unless `processingMode` is set explicitly in a test entry, `processingMode` is compatible with both `json-ld-1.0` and `json-ld-1.1`.
+
```

Comment:
processingMode probably isn't relevant, as there is no 1.0 mode for streaming processors.